README.md

```diff
@@ -1,118 +1,249 @@
+**TraceabilityCredential** | Credential for Tracability
+**ResiliencyCredential** | Credential for Use Case Resiliency
+**PcfCredential** | Credential for Use Case PCF
+
+**Example:**
+```
 {
-    "attribute_1": "value_1",
-    "attribute_2": "value_2",
-    ...
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        ...
+    ],    
+    "id": "UUID",
+    "issuer": "<issuerDID>",
+    "type": ["VerifiableCredential", "UseCaseFrameworkCondition"],    
+    "issuanceDate": "2021-06-16T18:56:59Z",
+    "expirationDate": "2022-06-16T18:56:59Z",    
+    "credentialSubject": {
+        "id": "<did>",
+        "holderIdentifier": "BPN of holder", 
+        "type": "SustainabilityCredential | BehaviorTwinCredential |QualityCredential ...", // UseCase Specific Entry
+        "contractTemplate": "https://public.catena-x.org/contracts/<UseCase>.v1.pdf",
+        "contractVersion": "1.0.0"   
+    }
 }
 ```
 
- 
+### Dismantler Schema
 
-...
+The Schema contains the Dismantler Credential Template
 
- 
+Schema Reference: [DismantlerSchema](./v1.1/DismantlerVC)
 
-## Data Types
+Property | Description
+--- | ---
+**id** | Id property that defines the Holder.
+**type** | MUST contain "MembershipCredential"
+**holderIdentifier** | Datetime indicating the start time of the membership.
+**allowedVehicleBrands** | Text indicating the group or organization the membership belongs to.
+**activityType** | Text indicating the status of the membership.
 
- 
+**Example:**
+```
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        ...
+    ],
+    "id": "UUID",
+    "issuer": "<did>",
+    "type": ["VerifiableCredential", "DismantlerCredential"],
+    "issuanceDate": "2021-06-16T18:56:59Z",
+    "expirationDate": "2022-06-16T18:56:59Z",
+    "credentialSubject": {
+        "id": "<did>",
+        "type": "DismantlerCredential", 
+        "holderIdentifier": "<BPN>",
+        "allowedVehicleBrands": ["CarBrand1", "CarBrand2", "CarBrand3"] 
+    }
+}
+```
 
-The context schema uses the following data types:
 
- 
+## Summary Schema
 
-- `string`: Represents textual data.
-- `integer`: Represents whole numbers.
-- `float`: Represents decimal numbers.
-- `boolean`: Represents true or false values.
-- `array`: Represents a collection of items.
-- `object`: Represents a complex data structure.
+The Summary Credentials is an Credential which sums up the Owned CX Credentials in one VC.
+
+Schema Reference: [SummarySchema](./v1.1/SummaryVC)
 
- 
+
+Property | Description
+--- | ---
+**id** | Id property that defines the contract template identity.
+**type** | MUST contain "SummaryCredential"
+**holderIdentifier** | Identifier of the holder (http://schema.org/identifier)
+**items** | List of items (https://schema.org/ItemList)
+**contractTemplate** | Text indicating the type of contract (https://schema.org/Text)
+
+---
+
+**Example**
+``` json
+ {
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    ...
```

Comment:
I think you want to delete this line

Reply:
I added this on purpose to show that more contexts are needed, should I let the context field completly Empty?